include/psen_scan_v2/ros_scanner_node.h

```diff
@@ -75,6 +79,10 @@ class ROSScannerNodeT
 
 private:
   void laserScanCallback(const LaserScan& scan);
+  void publishChangedIOStates(const std::vector<psen_scan_v2_standalone::IOState>& io_states);
+
+private:
+  static std::string formatPinState(const PinState& pin);
```

Comment:
The idea was not to make it part of the public API of the node. 
We could generate an anonymous or separate namespace to prevent this or move it somewhere else, but we decided to just put it in the class instead. What option would you prefer?

Reply:
Since this formats a PinState why not put it next to the definition of PinState. Or even in some dedicated logging function.